apps/mainsite/templates/account/email/email_confirmation_message.txt

```diff
@@ -1,8 +1,8 @@
-Bitte bestätige Deine Email-Adresse auf myBadges,
+Bitte bestätige Deine Email-Adresse auf openbadges,
 {{ email.email }}
 
-Ein*e myBadges Nutzer*in hat angefragt diese Email-Adresse zu ihrem/seinem Account hinzuzufügen oder sie in einer verknüpften Anwendung zu nutzen.
-Wenn bestätigt, kannst Du Badges zu Deinem myBadges Account hinzufügen, die zu dieser Adresse ausgestellt wurden oder features in verknüpften Anwendungen nutzen.
+Ein*e openbadges Nutzer*in hat angefragt diese Email-Adresse zu ihrem/seinem Account hinzuzufügen oder sie in einer verknüpften Anwendung zu nutzen.
```

Comment:
The naming is incosistent; sometimes it's "Open Badges", sometimes "openbadges".

Reply:
Open Educational Badges is too long, I thought we can use shortcut in some places. I only named it `Open Badges` in email title but now I think it's good to stick to one name, which we can change after discussion. Thanks Timber!